### PR TITLE
added select2 loading all tag, no sorting, no event handling

### PR DESCRIPTION
**Pull request by coolRaider**

### CODE REVIEW

File: app/controllers/tags_controller.rb [hunk 1]
```diff
@@ -9,7 +13,13 @@ def index
   end
 
   def show
+    @q = Article.ransack(params[:q])
     @tag = ActsAsTaggableOn::Tag.find(params[:id])
-    @articles = Article.tagged_with(@tag.name).paginate(:page => params[:page], :per_page => 4)
```
Comment (joney000) [inline app/controllers/tags_controller.rb]: per page only one article ?

File: app/controllers/tags_controller.rb [hunk 2]
```diff
@@ -9,7 +13,13 @@ def index
...
+    @articles = @q.result(distinct: true).tagged_with(@tag.name).paginate(:page => params[:page], :per_page => 1)
```
Comment (joney000) [inline app/controllers/tags_controller.rb:18]: per page only one article ?

File: app/views/articles/index.html.haml [hunk 3]
```diff
@@ -15,4 +15,4 @@
 
 = link_to 'New article', new_article_path
 
-= render "list", :articles => @articles
+#results= render "articles/list", :articles => @articles
```
Comment (joney000) [inline app/views/articles/index.html.haml:18]: line spacing. empty lines in between and no empty line at EOF.

File: app/controllers/tags_controller.rb (hunk 2 above)
Comment (coolRaider) [inline app/controllers/tags_controller.rb:18]: This is not up for review, work in progress and had to change it to 1 for testing as I don't have that many blogs in the system.

File: app/views/articles/index.html.haml (hunk 3 above)
Comment (coolRaider) [inline app/views/articles/index.html.haml:18]: single line space between sections was added intentionally for readability. EOF present on local copy, not sure why it is not being uploaded.

**Pull request closed without merging**